arbitrum-docs/for-devs/concepts/differences-between-arbitrum-ethereum/rpc-methods.mdx

```diff
@@ -0,0 +1,133 @@
+
+| Field name      | Description                                                                                                                                                                                          |
+| --------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `l1BlockNumber` | An approximate L1 block number that occurred before this L2 block. More information in [Block numbers and time](/for-devs/concepts/differences-between-arbitrum-ethereum/block-numbers-and-time.mdx) |
+| `sendCount`     | The number of L2 to L1 messages since Nitro genesis                                                                                                                                                  |
+| `sendRoot`      | The Merkle root of the outbox tree state                                                                                                                                                             |
+
+### Existing fields with different behavior
+
+On RPC calls that return a block, the following fields will have a different content than what's received on Ethereum.
+
+| Field name   | Description                                                                                                           |
+| ------------ | --------------------------------------------------------------------------------------------------------------------- |
+| `extraData`  | This field is equivalent to `sendRoot`                                                                                |
+| `mixHash`    | First 8 bytes is equivalent to `sendCount`, second 8 bytes is equivalent to `l1BlockNumber`                           |
+| `difficulty` | Fixed at `0x1`                                                                                                        |
+| `gasLimit`   | Value is fixed at `0x4000000000000`, but it's important to note that Arbitrum currently has a 32M gas limit per block |
+
+## Other methods that are slightly different
+
+### `eth_syncing`
+
+Calling `eth_syncing` returns false when the node is fully synced (just like on Ethereum). If the node is still syncing, `eth_syncing` returns an object with data about the synchronization status. Here, we provide more details.
+
+#### Understanding messages, batches, and blocks
+
+Nitro node reads messages from the parent chain an optionally from the message feed. It executes these messages and produces blocks. Each message produces exactly one block (a message may contain multiple transactions). In most Nitro chains, message number and block number are the same. However, Arbitrum One chain has pre-Nitro (classic) blocks, so for that chain, message 0 produced block `22207818`. Keep in mind that the offset between message and block number is constant in the chain.
```

Comment:
```suggestion
Nitro node reads messages from the parent chain and optionally from the message feed. It executes these messages and produces blocks. Each message produces exactly one block (a message may contain multiple transactions). In most Nitro chains, message number and block number are the same. However, Arbitrum One chain has pre-Nitro (classic) blocks, so for that chain, message 0 produced block `22207818`. Keep in mind that the offset between message and block number is constant in the chain.
```

Reply:
[Reworded the paragraph](https://nitro-docs-git-ethereum-vs-arbitrum-offchain-labs.vercel.app/for-devs/concepts/differences-between-arbitrum-ethereum/rpc-methods#understanding-messages-batches-and-blocks). Does it sound clearer/less confusing?